README.md

```diff
@@ -360,9 +362,7 @@ Ideally, we maintain a readable, clean, and linear commit history. To that end,
 >
 > NOTE: An interactive rebase (e.g., `git rebase -i`) can help you rewrite your branch's _local_ history to meet the criteria above
 
-# Other resources
-
-## Issues
+### Creating Issues
 
 New issues can be created in the Issues tab using the `New issue` button.
```

Comment:
Thanks for adding this bit to the guide. Can you also mention our Board as an alternative way to add Issues? Perhaps even the preferred way (while still listing both) since it will auto-fill the Project field and get devs used to this view over the Issues tab.

GitHub properly prompts for templates on Boards nowadays (it used to not do so).